### ISSUE TITLE
No longer plays any music since a recent update

### ISSUE THREAD

**Issue by daleao** [OMGDuke/SDH-GameThemeMusic#69]
I can't say which, exactly. Didn't touch my Deck for a few months, then came back and several things updated (OS, Client and Decky). One of those I suppose broke this plugin as I can no longer get any music to play at all.
Stable update channel for both OS (3.5.19) and Client, and prelease for DeckyLoader.

**Comment by Menojohnson** [comment]
Ive been having this issue with previous versions as well as with the latest updates too. hoping for an update soon.

**Comment by theMajc** [comment]
Same here.
Steam OS version 3.5.19 (stable)
Becky version 2.12.3 (stable)
Game theme music 1.5.0

**Comment by ThisByzantineConduit** [comment]
Seems to be an issue with the default official Piped Instance (the one that uses multiple countries). If you go to search for songs with that you'll see it initially populate a big list but then every single one will disappear. Not sure what causes this.

For now you can try another Piped Instance that has the songs you want. Sucks because the others are down more often and I have to keep switching between them. Not sure why the official one shows as being up but can't pull a single song. Hopefully there's a fix soon 🤞.

**Comment by daleao** [comment]
Changing the Piped instance works. Unfortunately most of them take a long ass time to start playing, at least in my case as someone not living in Europe.

**Comment by ThisByzantineConduit** [comment]
> Changing the Piped instance works. Unfortunately most of them take a long ass time to start playing, at least in my case as someone not living in Europe.

Yeah, noticed this in USA as well but for me it's inconsistent. Sometimes it'll start playing immediately but other times it can be delayed by as much as 10-20 seconds and sometimes I even need to exit the game page and reload.

I've also noticed another issue where with any Piped Instance, the music will sometimes skip around to other points in the track. Its weird because I used this plugin for a long time without any of these issues. Maybe it's an issue with SteamOS updates 🤔.

**Comment by ThisByzantineConduit** [comment]
This is related to the same issue that others have created issue tickets here on, which is the song list not populating; or rather, when you search for a song it shows tons of results but then all those populated results vanish. This is why no music plays as well.

This has been happening for a while, but I've been able to circumvent it by choosing an alternate Piped Instance until now. For years this plugin worked flawlessly but now the default/official international Piped Instance and all alternative U.S.A. Piped Instances, and many from other regions, populate a big list of songs which then completely disappears leaving no music options.

This is consistent no matter how generalized my search terms are. Any idea on a fix? Tried reinstalling the plugin to no avail.
